src/Uno.UI.sln

```diff
@@ -219,6 +219,7 @@ Global
 		SamplesApp\SamplesApp.Shared\SamplesApp.Shared.projitems*{313fcf91-a6db-4585-a23e-49bbeef4784d}*SharedItemsImports = 4
```

Comment:
Should this be included?

Reply:
It's curious that it's not. Could be a vs4mac change ?

Reply:
Using VS4M, no idea how that file got modified. I removed that line

cc: @jeromelaban  @davidjohnoliver